book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+  </tr>
+  <tr>
+   <td>Investing in Women
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://investinginwomen.co.uk/">https://investinginwomen.co.uk/</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Girls in Tech (US)
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://girlsintec.com/opportunities">https://girlsintec.com/opportunities</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Girls who code (UK)
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://girlswhocode.com/en-uk">https://girlswhocode.com/en-uk</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Mothers in Science
+   </td>
+   <td>Women/Mothers
+   </td>
+   <td><a href="https://www.mothersinscience.com/">Mothers in Science</a>
+   </td>
+  </tr>
+  <tr>
+   <td>PyLadies
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.pyladies.com/">https://www.pyladies.com/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>R Ladies
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://rladies.org/">https://rladies.org/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>STEM women
+   </td>
+   <td>Women 
+   </td>
+   <td><a href="https://www.stemwomen.com/job-search">https://www.stemwomen.com/job-search</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Technovation
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://technovationchallenge.org/get-started/">https://technovationchallenge.org/get-started/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Tech Returners
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.womenintech.co.uk/events/techreturners">https://www.womenintech.co.uk/events/techreturners</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>TeenTech
+   </td>
+   <td>Women
+   </td>
+   <td><a href="http://www.teentech.com/">http://www.teentech.com/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>WISE Campaign
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.wisecampaign.org.uk">https://www.wisecampaign.org.uk</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Women in AI
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.womeninai.co/">https://www.womeninai.co/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Women in Data
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.meetup.com/Women-in-Data/">https://www.meetup.com/Women-in-Data/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Women in Big Data
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.womeninbigdata.org/">https://www.womeninbigdata.org/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Women in Data Science
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.widsworldwide.org/">https://www.widsworldwide.org/</a> 
+   </td>
+  </tr>
+  <tr>
+   <td>Women in Data Science and Machine Learning
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.linkedin.com/groups/8466901/">https://www.linkedin.com/groups/8466901/</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Women in Tech
+   </td>
+   <td>Women
+   </td>
+   <td><a href="https://www.womenintech.co.uk/">https://www.womenintech.co.uk/</a> 
+   </td>
+  </tr>
+</table>
```

Comment:
Women in HPC: https://womeninhpc.org/community/jobs

Reply:
Please add this to issue #4149 :)